test/e2e/e2e.go

```diff
@@ -69,6 +74,18 @@ func TearDown(clients *test.Clients, cleaner *test.Cleaner, _ logging.FormatLogg
 	cleaner.Clean(true)
 }
 
+// CreateRandomSubscriptionNames will create random number of subscription names
```

Comment:
I'm typically not a fan of adding randomness to the tests unless there's a good reason for it. What are we testing with this?

Reply:
I was thinking having random number of subscriptions may help us to cover more scenarios. But now I realized it may make it harder to debug when there is a problem. I have removed this random logic.